plugins/generic/googleScholar/GoogleScholarPlugin.inc.php

```diff
@@ -0,0 +1,138 @@
+	 * @return boolean
+	 */
+	function articleView($hookName, $args) {
+		$request = $args[0];
+		$issue = $args[1];
+		$article = $args[2];
+		$journal = $request->getContext();
+
+		$templateMgr = TemplateManager::getManager($request);
+		$templateMgr->addHeader('googleScholarRevision', '<meta name="gs_meta_revision" content="1.1"/>');
+		$templateMgr->addHeader('googleScholarJournalTitle', '<meta name="citation_journal_title" content="' . htmlspecialchars($journal->getLocalizedName()) . '"/>');
+		if (($issn = $journal->getSetting('onlineIssn')) || ($issn = $journal->getSetting('printIssn')) || ($issn = $journal->getSetting('issn'))) {
+			$templateMgr->addHeader('googleScholarIssn', '<meta name="citation_issn" content="' . htmlspecialchars($issn) . '"/> ');
+			
+		}
+
+		foreach ($article->getAuthors() as $i => $author) {
+			$templateMgr->addHeader('googleScholarAuthor' . $i, '<meta name="citation_author" content="' . htmlspecialchars($author->getFirstName()) . (($middleName = htmlspecialchars($author->getMiddleName()))?" $middleName":'') . ' ' . htmlspecialchars($author->getLastName()) . '"/>');
+			if ($affiliation = htmlspecialchars($author->getLocalizedAffiliation())) {
+				$templateMgr->addHeader('googleScholarAuthor' . $i . 'Affiliation', '<meta name="citation_author_institution" content="' . $affiliation . '"/>');
+			}
+		}
+
+		$templateMgr->addHeader('googleScholarTitle', '<meta name="citation_title" content="' . htmlspecialchars($article->getLocalizedTitle()) . '"/>');
+
+		if (is_a($article, 'PublishedArticle') && ($datePublished = $article->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		} elseif ($issue && $issue->getYear()) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . htmlspecialchars($issue->getYear()) . '"/>');
+		} elseif ($issue && ($datePublished = $issue->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		}
+
+		if ($issue) {
+			$templateMgr->addHeader('googleScholarVolume', '<meta name="citation_volume" content="' . htmlspecialchars($issue->getVolume()) . '"/>');
+			$templateMgr->addHeader('googleScholarNumber', '<meta name="citation_issue" content="' . htmlspecialchars($issue->getNumber()) . '"/>');
+		}
+
+		if ($article->getPages()) {
+			if ($startPage = $article->getStartingPage()) $templateMgr->addHeader('googleScholarStartPage', '<meta name="citation_firstpage" content="' . htmlspecialchars($startPage) . '"/>');
+			if ($endPage = $article->getEndingPage()) $templateMgr->addHeader('googleScholarEndPage', '<meta name="citation_lastpage" content="' . htmlspecialchars($endPage) . '"/>');
+		}
+
+		foreach($templateMgr->get_template_vars('pubIdPlugins') as $pubIdPlugin) {
+			if ($pubId = $pubIdPlugin->getPubId($article)) {
```

Comment:
Ah -- I brought this in directly from the Dublin Core / Google Scholar templates. Is there an example elsewhere in the code for how this should look?